css/details.css

```diff
@@ -133,3 +133,7 @@ body {
   gap: 8px;
   flex-wrap: wrap;
 }
+
+.loader.hide {
+  display: none;
+}
```

Comment:
Do you need the `.loader` selector? will using plain `.hide` work ? 

Also I would call it `hidden`